lib/Common/ConfigFlagsList.h

```diff
@@ -679,6 +679,7 @@ PHASE(All)
 #define DEFAULT_CONFIG_ES6RegExSticky          (true)
 #define DEFAULT_CONFIG_ES2018RegExDotAll       (true)
 #define DEFAULT_CONFIG_ESBigInt                (false)
+#define DEFAULT_CONFIG_ESNumericSeparator      (true)
```

Comment:
Thanks for defaulting this to true. : )

At some point I think we'll need to comb through some of the old always-on feature flags and clean them up.

Reply:
Yeah. We've made passes before and took out some of the ancient flags for things which are never getting disabled. Worth looking at again.